src/render.tsx

```diff
@@ -111,13 +111,17 @@ function buildRenderResult(
       return getHostChildren(instance)[0];
     },
     UNSAFE_root: instance,
-    get container(): ReactTestInstance {
+  };
+
+  Object.defineProperty(result, 'container', {
+    enumerable: false,
```

Comment:
Worth adding a comment explaining the reasoning? I guess it won't be obvious for most people, on the other end, we have a pretty explicit test for that.

Reply:
Added comment for that